src/pages/transactions/_components/DfTx/DfTxCompositeSwap.tsx

```diff
@@ -153,13 +186,24 @@ function PoolTo({
         <AdaptiveList.Row name="Token" testId="DfTxCompositeSwap.TokenTo">
           {ToTokenSymbol}
         </AdaptiveList.Row>
-        <AdaptiveList.Row name="Max Amount" testId="DfTxCompositeSwap.MaxPrice">
+        <AdaptiveList.Row name="Max Price" testId="DfTxCompositeSwap.MaxPrice">
           <NumberFormat
             value={poolswap.maxPrice.toFixed(8)}
             thousandSeparator
             displayType="text"
           />
         </AdaptiveList.Row>
+        <AdaptiveList.Row name="Amount" testId="DfTxCompositeSwap.toAmount">
+          {toAmount === undefined ? (
+            <div>pending</div>
```

Comment:
What is this pending?

Reply:
For raw txn we are placing `pending` as raw txn dont have any history so we wont be able to get to amount